src/js/views/forms/form/form_views.js

```diff
@@ -34,9 +34,14 @@ const ContextTrailView = View.extend({
     'click .js-dashboard': 'click:dashboard',
   },
   onClickBack() {
-    Radio.request('history', 'go:back');
+    Radio.request('history', 'go:back', () => {
+      this.routeToPatient();
+    });
```

Comment:
It's not letting me comment on that line, but does the go back in the `PreviewView` in this file need a `handleEmptyHistory` function as well?

This line here: https://github.com/RoundingWell/care-ops-frontend/pull/944/commits/01d2176d7336ef6d2ce2b1e0772d6f6459ae4b6e#diff-e2f36acf877614e07a231a6f78b71f617ab8c1cc839269088dad7a7469d74a0bR229

Reply:
Not sure where we'd go to necessarily.  I guess we could potentially go back to the program, but TBH the form preview is pretty broken.. it can't load context in a way that most forms are probably pretty broken to preview.

Reply:
Okay, only thing I noticed. Things are working nicely locally.